capi-configuration/1_default-customization.md

```diff
@@ -0,0 +1,29 @@
+# Sharing responsibility and customization with customers
+
+**THIS IS WIP AND LATER IN THE FUTURE**
+
+## Problem Statement
+
+We want to ...
+1. give customers greater control over how their clusters are defaulted.
+2. enable customers to have more agency in upgrading individual components.
+
+## Enabling customer access
+
+We have to be able to give customers access to the `git` source in a gitops setup in order to allow effective modification of any configuration.
+
+We already have a shared repository in the Giant Swarm organization with each customer individually.
```

Comment:
So, we'd use each customer's repo as a source for gitops data. How do we synchronise across customer repos?